### PR TITLE
docs: Alert component documentation re-organization

### PR DESCRIPTION
**Pull request by osbornedesign**
image

### CODE REVIEW

File: src/components/Alert/Alert.VisualTests.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    ))}
+  </Box>
+);
+
+export const VariantsWithMessage = Template.bind({});
+VariantsWithMessage.args = {
+  message: 'Hello world!',
+}
+
+export const WithMessageWithIcon = Template.bind({});
```
Comment (nathanyoung) [inline src/components/Alert/Alert.VisualTests.stories.tsx:42]: You can add a more human readable storyname like this for each component:

`WithMessageWithIcon.storyName = 'With Message, Icon";`

https://storybook.js.org/docs/react/api/csf#named-story-exports

File: src/components/Alert/Alert.VisualTests.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+  isClosable: true,
+}
+
+export const WithMessageWithIconWithClosable = Template.bind({});
+WithMessageWithIconWithClosable.args = {
+  message: 'Hello world!',
+  isClosable: true,
+  hasIcon: true,
+}
+
```
Comment (nathanyoung) [inline src/components/Alert/Alert.VisualTests.stories.tsx:60]:
```suggestion
WithMessageWithIconWithClosable.storyName = "With Message, Icon, Closable"
```

File: src/components/Alert/Alert.Playground.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { Meta, Story } from '@storybook/react/types-6-0';
+import Alert, { AlertProps } from './Alert';
+
+const alertVariants = [
```
Comment (nathanyoung) [inline src/components/Alert/Alert.Playground.stories.tsx:5]: Use the `AlertVariants` defined in Alert.tsx instead of creating the same thing here.

**Review by nathanyoung — CHANGES_REQUESTED** [review]

**Review by juanfabrega — APPROVED** [review]
This is some good work!

**Review by nathanyoung — APPROVED** [review]

**Pull request merged**